controllers/blogController.js

```diff
@@ -10,10 +10,12 @@ const blogs_get = async (req, res) => {
   }
 }
 
-const blogs_single_get = async (req, res) => {
+// Not working yet. Working on it.
+const blog_get = async (req, res) => {
+  const id = req.params.id;
     try {
       const blogs = await Blog.findById(id);
```

Comment:
You may want to call this variable blog (singular), as the Blog.findById() returns a single blog.